libs/csharp.json

```diff
@@ -75,5 +75,7 @@
   {"id": "cs.fody", "imports": ["Fody"], "name": "Fody", "repo": "Fody/Fody", "tags": ["IL-rewrite"], "tech": ["eng-tools"], "status": "awaiting-model"},
   {"id": "cs.protobuf-net", "imports": ["ProtoBuf"], "name": "ProtoBuf.NET", "repo": "mgravell/protobuf-net", "tags": ["serialization"], "tech": ["efficient-data"], "status": "awaiting-model"},
   {"id": "cs.serilog", "imports": ["Serilog"], "name": "Serilog", "repo": "serilog/serilog", "tags": ["logging"], "tech": ["diagnostics"], "status": "awaiting-model"},
-  {"id": "cs.castle-windsor", "imports": ["Castle"], "name": "castle.windsor", "repo": "castleproject/Windsor", "tags": ["inversion-of-control"], "tech": ["app-arch"], "status": "model-ready"}
+  {"id": "cs.castle-windsor", "imports": ["Castle"], "name": "castle.windsor", "repo": "castleproject/Windsor", "tags": ["inversion-of-control"], "tech": ["app-arch"], "status": "model-ready"},
+  {"id":"cs.boost","imports":["boost"],"name":"boost","repo":"sergey-shandar/getboost","tags":["csharp-boost"],"tech":["interoperability"],"status":"awaiting-model"},
```

Comment:
This is very strange. Boost is a C++ library. This repo is NuGet packages for Boost. Basically, it's just a way for C++ programmers in Visual Studio to pull their Boost.

Reply:
will remove it